### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tristandl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,17 +167,20 @@ checkout:
...
+    - bin/execute.sh test
     - mv build/reports/* $CIRCLE_TEST_REPORTS
     - cp -r $CIRCLE_TEST_REPORTS $CIRCLE_ARTIFACTS
 deployment:
   itunes_connect:
-    branch: release
+    branch: master
     commands:
-      - chruby 2.3.1 && bin/execute.sh itunes-connect --scheme "MyProject-Prod"
+      - bin/execute.sh itunes-connect --scheme "MyProject-Prod"
```
Comment (aexmachina) [inline README.md]: I don't think you want to deploy prod off `master`! Also, I've been using release branches named `release/production` and `release/beta`, I think it's a good option.

Comment (aexmachina) [inline README.md]: Otherwise 🚀

Comment (tristandl) [inline README.md]: ok i'll put that back. i like the branch folder idea. i also want to make a couple of changes to how the schemes work with alpha/beta/prod and internal testers/external testers/app store as well as those suffix/icons on the app icon.

Comment (tristandl) [inline README.md]: MT doesn't have a separate release branch though - it never did.

**Pull request merged**